tests/pds/dataservice_test.go

```diff
@@ -758,17 +761,6 @@ func deployAndTriggerTpcc(dataservice, Version, Image, dsVersion, dsBuild string
 				}
 			}
 		})
-
-		Step("Delete Deployments", func() {
```

Comment:
removing this part will delete only the latest deployment

Reply:
Done